tests/Diggin/RobotRules/Accepter/TxtGoogleWebmasterTest.php

```diff
@@ -22,7 +22,7 @@ public function testCommentAndUseragent()
 $txt = <<<EOF
 User-agent: Googlebot-Mobile
 Disallow: /path/
-# aa
+
```

Comment:
Ok! remove.

This comment line was intended added as ignorable.
(sorry for non-clear comment)

Basic Test (TxtTest.php) already checks comment-only line.
So, adding in this test was not meaningful.